mpas_analysis/analysis_task_template.py

```diff
@@ -0,0 +1,217 @@
+'''
+This is an example analysis task to be used as a template for new tasks.
+It should be copied into one of the component folders (`ocean`, `sea_ice`,
+`land_ice`, etc.) and modified as needed.
+
+Specific parts or variable names that need to be modified by the developer are
+explicitly indicated. All the others are to be left unchanged.  Anything in
+comments or docstrings in triangle brackets <> indicates that it needs to be
+replaced with the appropriate variable name, text, etc. for the task.
+
+Instructions for creating a new analysis task:
+1. create a new task by copying this file to the appropriate folder (ocean,
+   sea_ice, etc.) and modifying it as described below.  Take a look at
+   mpas_analysis/shared/analysis_task.py for additional guidance.
+2. modify config.default (and possibly any machine-specific config files in
+   configs/<machine>)
+3. import new analysis task in mpas_analysis/<component>/__init__.py
+4. note, no changes need to be made to mpas_analysis/shared/analysis_task.py
```

Comment:
Thanks for adding these steps here as well, @xylar.
Maybe I would move 4. to 2. (right after 'Take a look..', so that it is clear that analysis_task can be referenced to, but doesn't have to be modified).